constraint_utils/constraint_basics.py

```diff
@@ -704,7 +771,22 @@ def build_plateau_models(ht: hl.Table, weighted: bool = False) -> Dict[str, Tupl
 
 def build_plateau_models_pop(ht: hl.Table, weighted: bool = False) -> Dict[str, Tuple[float, float]]:
```

Comment:
This function raised `ClassTooLargeException`. Should I fix the function here or in the new constraint repo?

Reply:
Let's leave as is here and fix in the new repo